libs/SalesforceSDKCore/SalesforceSDKCoreTests/SalesforceRestAPITests.m

```diff
@@ -314,7 +314,7 @@ - (void)testCreateQuerySearchDelete {
 // - query updated object (requestForSearch) and make sure we don't get anything
 - (void)testCreateUpdateQuerySearchDelete {
     // create object
-    NSString *lastName = [NSString stringWithFormat:@"Doe-%@", [NSDate date]];
```

Comment:
But shouldn't we correct that issue?  There's no reason there can't be a '+' in the query, right?